### PR TITLE
migrate `AIK v0.2.0`

### PR DESCRIPTION
**Pull request by ktym4a**

### CODE REVIEW

**Comment by ktym4a** [comment]
I think the migration should be correct, cuz it is working correctly.

@florian-lefebvre Can you review it, when you have time?

File: package/src/astro-page-insight.ts [hunk 1]
```diff
@@ -1,18 +1,17 @@
...
 	name: "astro-page-insight",
 	plugins: [...corePlugins],
-	options: defineOptions<Options>({ weight: 0, breakPoint: 768 }),
+	optionsSchema: z.object({
+		weight: z.number().optional().default(0),
+		breakPoint: z.number().optional().default(768),
+	}),
 	setup({ options: inputOptions }) {
-		const { weight = 0, breakPoint = 768 } = optionsSchema.parse(inputOptions);
+		const { weight, breakPoint } = inputOptions;
```
[the remark this answers is not in the document]
Comment (ktym4a) [reply package/src/astro-page-insight.ts]: updated!

File: package/src/astro-page-insight.ts [hunk 2]
```diff
@@ -1,18 +1,17 @@
...
 import { corePlugins } from "astro-integration-kit/plugins";
+import { z } from "astro/zod";
 import { organizeLHResult, startLH } from "./server.js";
-import { type Options, optionsSchema } from "./types.js";
 
 export default defineIntegration({
 	name: "astro-page-insight",
 	plugins: [...corePlugins],
-	options: defineOptions<Options>({ weight: 0, breakPoint: 768 }),
+	optionsSchema: z.object({
```
[the remark this answers is not in the document]
Comment (ktym4a) [reply package/src/astro-page-insight.ts:9]: added!

File: package/src/astro-page-insight.ts [hunk 3]
```diff
@@ -1,18 +1,28 @@
...
 	plugins: [...corePlugins],
-	options: defineOptions<Options>({ weight: 0, breakPoint: 768 }),
-	setup({ options: inputOptions }) {
-		const { weight = 0, breakPoint = 768 } = optionsSchema.parse(inputOptions);
+	optionsSchema: z.object({
+		/**
+		 * `weight` is the threshold value in the audit.
+		 * All audit items have weights assigned by lighthouse and can be filtered by thresholds(`weight`).
+		 *
+		 * @default 0
```
Comment (florian-lefebvre) [inline package/src/astro-page-insight.ts:14]:
```suggestion
		 * @default `0`
```

File: package/src/astro-page-insight.ts [hunk 4]
```diff
@@ -1,18 +1,28 @@
...
+		 */
+		weight: z.number().optional().default(0),
+		/**
+		 * `breakPoint` is used to determine whether on mobile or desktop.
+		 * if the viewport width is less than the `breakPoint`, the lighthouse will run as a mobile device.
+		 *
+		 * @default 768
```
Comment (florian-lefebvre) [inline package/src/astro-page-insight.ts:21]:
```suggestion
		 * @default `768`
```

**Review by florian-lefebvre — COMMENTED** [review]
Little knit

**Comment by ktym4a** [comment]
Thanks for your review again!
I approved and will merge it.

**Pull request merged**